### PR TITLE
done some translation!-kingda

### PR DESCRIPTION
**Pull request by KingdaShi**

### CODE REVIEW

**Comment by MichaelChirico** [comment]
LGTM! @Zachary-Wu please take a look

File: po/R-zh_CN.po [hunk 1]
```diff
@@ -1527,86 +1527,86 @@ msgid "the regex \""
 msgstr "正则 \""
 
 msgid "\". Please double check the input file is a valid csvy."
-msgstr ""
```
Comment (Zachary-Wu) [inline po/R-zh_CN.po]: "\". 请再次检查输入文件是否为有效的csvy。 < CSVY is a file format that combines the simplicity of CSV (comma-separated values) with the metadata of other plain text and binary formats (JSON, XML, Stata, etc.) by placing a YAML header on top of a regular CSV. >

File: po/R-zh_CN.po [hunk 2]
```diff
@@ -1527,86 +1527,86 @@ msgid "the regex \""
...
 
 msgid "]); the proceeding assumes the user input was"
-msgstr ""
+msgstr "]); 该过程假定用户输入的是"
 
 msgid ""
 "an intentional override and will ignore the types implied by the YAML "
 "header; please exclude"
-msgstr ""
+msgstr "有意覆盖的，并且将忽略YAML列名所指示的类型；"
```
Comment (Zachary-Wu) [inline po/R-zh_CN.po]: "有意覆盖的，因此将忽略YAML表头所指示的列类型。"

File: po/zh_CN.po [hunk 3]
```diff
@@ -2987,253 +2987,256 @@ msgstr ""
...
-"Z_FINISH=%d Z_BLOCK=%d. %s"
-msgstr ""
+msgid "zlib v%s deflate() returned error %d with z_stream.msg '%s'. %s\n"
+msgstr "zlib v%s deflate()使用%d with z_stream.msg '%s'. %s返回了错误\n"
 
-#: fwrite.c:967
-msgid ""
-"Please include the full output above and below this message in your data."
-"table bug report."
-msgstr ""
```
[the remark this answers is not in the document]
Comment (MichaelChirico) [reply po/zh_CN.po]: please have another look, the message in English also changed slightly

[the remark this answers is not in the document]
Comment (Zachary-Wu) [reply po/zh_CN.po]: so, I want to know the meaning of include. if it is a word which means add something as a part of another thing or a program directive.

[the remark this answers is not in the document]
Comment (MichaelChirico) [reply po/zh_CN.po]: basically, if you see this error message, we want you to copy-paste all of the output in your R session from the latest command into a bug report, which will help us diagnose the issue, does that make sense?

File: po/zh_CN.po [hunk 4]
```diff
@@ -3027,254 +3027,258 @@ msgstr ""
...
-"Z_FINISH=%d Z_BLOCK=%d. %s"
-msgstr ""
+msgid "zlib v%s deflate() returned error %d with z_stream.msg '%s'. %s\n"
+msgstr "zlib v%s deflate()使用%d with z_stream.msg '%s'. %s返回了错误\n"
 
 #: fwrite.c:966
 msgid ""
 "Please include the full output above and below this message in your data."
 "table bug report."
-msgstr ""
```
Comment (Zachary-Wu) [inline po/zh_CN.po]: "如果您看到这条消息，请将此消息上方和下方的全部输出结果一起提交到data.table的bug report 中，我们将进一步诊断和处理该bug。"

**Comment by MichaelChirico** [comment]
@KingdaShi also please have a look at the suggestions of DeLun here 😄

**Comment by MichaelChirico** [comment]
LGTM, great work @kingdashi @zachary-wu !!

**Pull request merged**